docker/py3-tools/Pipfile

```diff
@@ -21,20 +20,18 @@ imapclient = "*"
 pyjarm = "*"
 pyminizip = "*"
 paho-mqtt = "*"
-arrow = "*"
 pytz = "*"
 python-dateutil = "*"
 vertica-python = "*"
 sendgrid = "*"
-circleci = "*"
 rsa-archer = "*"
 tweepy = "*"
 tld = "*"
 validate-email = "*"
 requests = "*"
 markdownify = "*"
 langdetect = "*"
-gql = "*"
+gql = "==3.0.0rc0"
```

Comment:
we should be careful with rc,  for now as for test propose we can have it

Reply:
I chancged to the only used content usage of the graphql image (gql image isn't used) and graphql image uses the rc version. What do you say?